client/landing/gutenboarding/onboarding-block/style-preview/style.scss

```diff
@@ -40,17 +41,41 @@
 
 .style-preview__preview {
 	grid-area: preview;
-
-	border: 1px solid var( --studio-gray-5 );
-	box-shadow: 0 4px 14px rgba( 0, 0, 0, 0.25 );
+	margin: 0 auto;
+	background: var( --studio-white );
+	box-shadow: 0 0 0 1px var( --studio-gray-5 ), 0 4px 14px rgba( 0, 0, 0, 0.25 );
 	border-radius: 4px;
 	overflow: hidden;
+	width: 100%;
+
+	&.is-viewport-mobile,
+	&.is-viewport-tablet {
+		width: 375px;
+		height: 812px;
+		box-sizing: border-box;
+		border: 13px solid var( --studio-white );
+
+		box-shadow: 0 0 0 1px var( --studio-gray-5 ), 0 4px 14px rgba( 0, 0, 0, 0.14 ),
+			inset 0 0 0 1px var( --studio-gray-5 );
+		border-radius: 31px;
+	}
+
+	&.is-viewport-tablet {
+		width: 1024px;
+		height: 768px;
+	}
+
+	&.is-viewport-mobile {
+		width: 351px;
+		height: 690px;
+	}
 }
+
+// This and the following dot class render the browser chrome
 .style-preview__preview-bar {
 	width: 100%;
 	height: 30px;
-
-	background: #fff;
+	background: var( --studio-white );
```

Comment:
Are we decided on using `--studio..` vs `--color..` or is this something that doesn't exist in both? I've seen some chatter on Slack about this, would love to know. :-)

Reply:
We probably have a mixture of things now since it's been unclear. :-) It's ok for this PR to go as-is I think but we'll need to consolidate at some point. Opened an issue to track: https://github.com/Automattic/wp-calypso/issues/40316